static/skywire-manager-src/src/app/components/vpn/pages/vpn-server-list/vpn-server-list.component.ts

```diff
@@ -0,0 +1,877 @@
+  latencyRatingSortData = new SortingColumn(['latencyRating'], 'vpn.server-list.latency-rating-small-table-label', SortingModes.Number);
+  hopsSortData = new SortingColumn(['hops'], 'vpn.server-list.hops-small-table-label', SortingModes.Number);
+  noteSortData = new SortingColumn(['note'], 'vpn.server-list.note-small-table-label', SortingModes.Text);
+
+  private dataSortedSubscription: Subscription;
+  private dataFiltererSubscription: Subscription;
+  // Objects in charge of sorting and filtering the data.
+  dataSorter: DataSorter;
+  dataFilterer: DataFilterer;
+
+  // If the server list is being loaded.
+  loading = true;
+  // If the app is still loading the data about the local visor. Must be true for showing the list.
+  loadingBackendData = true;
+  // Data for populating the list.
+  dataSource: VpnServerForList[];
+  // Data for populating the tabs of the top bar.
+  tabsData = VpnHelpers.vpnTabsData;
+
+  // Vars for the pagination functionality.
+  allServers: VpnServerForList[];
+  filteredServers: VpnServerForList[];
+  serversToShow: VpnServerForList[];
+  numberOfPages = 1;
+  currentPage = 1;
+  // Used as a helper var, as the URL is read asynchronously.
+  currentPageInUrl = 1;
+
+  // Pk of the local visor.
+  currentLocalPk: string;
+  // List currently being shown. It also means which tab is currently selected in the lower tab bar.
+  currentList = Lists.Public;
+  // Currently selected server.
+  currentServer: LocalServerData;
+  // If the VPN is currently running.
+  vpnRunning = false;
+
+  serverFlags = ServerFlags;
+  lists = Lists;
+
+  // Array with the properties of the columns that can be used for filtering the data.
+  filterProperties: FilterProperties[];
+
+  private initialLoadStarted = false;
+
+  private navigationsSubscription: Subscription;
+  private dataSubscription: Subscription;
+  private currentServerSubscription: Subscription;
+  private backendDataSubscription: Subscription;
+  private checkVpnSubscription: Subscription;
+
+  constructor(
+    private dialog: MatDialog,
+    private router: Router,
+    private translateService: TranslateService,
+    private route: ActivatedRoute,
+    private vpnClientDiscoveryService: VpnClientDiscoveryService,
+    private vpnClientService: VpnClientService,
+    private vpnSavedDataService: VpnSavedDataService,
+    private snackbarService: SnackbarService,
+  ) {
+    this.navigationsSubscription = route.paramMap.subscribe(params => {
+      // Get which list must be shown.
+      if (params.has('type')) {
+        if (params.get('type') === Lists.Favorites) {
+          this.currentList = Lists.Favorites;
+          this.listId = 'vfs';
+        } else if (params.get('type') === Lists.Blocked) {
+          this.currentList = Lists.Blocked;
+          this.listId = 'vbs';
+        } else if (params.get('type') === Lists.History) {
+          this.currentList = Lists.History;
+          this.listId = 'vhs';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
          this.listId = "vhs";
```